exporter/awscloudwatchlogsexporter/factory.go

```diff
@@ -46,15 +46,23 @@ func createLogsExporter(_ context.Context, set component.ExporterCreateSettings,
 		return nil, errors.New("invalid configuration type; can't cast to awscloudwatchlogsexporter.Config")
 	}
 
+	if !oCfg.QueueSettings.Enabled {
+		return nil, errors.New("invalid configuration; sending_queue.enabled must be set to true")
+	}
+	if oCfg.QueueSettings.QueueSize < 1 {
+		return nil, errors.New("invalid configuration; sending_queue.queue_size must be 1 or greater")
+	}
+	// due to the sequence token, there can be only one request in flight
+	if oCfg.QueueSettings.Consumers != 1 {
+		return nil, errors.New("invalid configuration; sending_queue.num_consumers must be set to 1")
+	}
```

Comment:
Add this in verify for config

Reply:
But is it legit? I kind of wish there was a way to check if this was set or not at all, which would require using `*QueueSettings`, and that's not something I can see in other exporters, so didn't dare to introduce... also I'm not familiar with `mapstructure` parser either.

Reply:
I mean, I'd want to have e.g.
```Go
if oCfg.QueueSettings == nil {
   oCfg.QueueSettings = defaultQueueSettings()
}
```

Reply:
I mean on the "Config" struct need a `Validate` func:

https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/exporter.go#L56

Reply:
Yes, sure. What I'm asking is whether the logic is actually sensible for this purpose, I wish it had defaults instead of asking the user to provide correct configuration...